### PR TITLE
Set standard to use authenticatable per Devise's coding standard.

### LINKED ISSUE #45: authenticable vs authenticatable
### Question
Should we have a documentation/code naming standard for authenticable/authenticatable? 

authenticatable has 14 results with only one in the documentation.
authenticable has 38 results with five in the documentation.

I did some googling and these mean the same thing with authenticable referencing authenticatable as being an alternate form of it (meaning authenticatable is the more standard form of this word). See:
https://en.wiktionary.org/wiki/authenticable

What's more is that Devise itself addresses this issue and the decision was to conform to authenticatable:

https://github.com/plataformatec/devise/issues/4157

If we were to move to authenticatable this would be a breaking change so it would probably be good to do it as soon as possible before this library gets really popular.

### PR DESCRIPTION
**Pull request by aarona**
Resolves #45

### CODE REVIEW

File: lib/graphql_devise/rails/routes.rb [hunk 1]
```diff
@@ -39,9 +39,9 @@ def mount_graphql_devise_for(resource, opts = {})
         skip:   [:sessions, :registrations, :passwords, :confirmations, :omniauth_callbacks, :unlocks]
       )
 
-      authenticable_type = opts[:authenticable_type] ||
+      authenticatable_type = opts[:authenticatable_type] ||
                            "Types::#{resource}Type".safe_constantize ||
-                           GraphqlDevise::Types::AuthenticableType
+                           GraphqlDevise::Types::AuthenticatableType
```
Comment (mcelicalderon) [inline lib/graphql_devise/rails/routes.rb]: Please align this and the line above with `opts`

File: spec/dummy/config/routes.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
   mount_graphql_devise_for(
     'Admin',
-    authenticable_type: Types::CustomAdminType,
+    authenticatable_type: Types::CustomAdminType,
     skip:               [:sign_up, :check_password_token],
     at:                 '/api/v1/admin/graphql_auth'
```
Comment (mcelicalderon) [inline spec/dummy/config/routes.rb]: This and the above need to be aligned too

File: lib/graphql_devise/rails/routes.rb (hunk 1 above)
Comment (aarona) [reply lib/graphql_devise/rails/routes.rb]: Got it. Not sure if this conforms to your styling preferences but what about this:
```ruby
      authenticatable_type =
        opts[:authenticatable_type] ||
        "Types::#{resource}Type".safe_constantize ||
        GraphqlDevise::Types::AuthenticatableType
```

Comment (mcelicalderon) [reply lib/graphql_devise/rails/routes.rb]: Our current rubocop config prefers the other approach

File: spec/dummy/config/routes.rb (hunk 2 above)
Comment (aarona) [reply spec/dummy/config/routes.rb]: Done.

**Comment by aarona** [comment]
@00dav00 thanks! Check out my comment I just made on a closed issue I made regarding rubocop btw.

File: lib/graphql_devise/rails/routes.rb (hunk 1 above)
Comment (aarona) [reply lib/graphql_devise/rails/routes.rb]: Got it. I'll make the proper adjustment and commit the change.

**Pull request merged**